Source/CarthageKit/Xcode.swift

```diff
@@ -172,7 +172,12 @@ public struct BuildArguments {
 		// Disable code signing requirement for all builds
 		// Frameworks get signed in the copy-frameworks action
 		args += [ "CODE_SIGNING_REQUIRED=NO", "CODE_SIGN_IDENTITY=" ]
-
+		
+		// If we're in xcode phase, sanitize
+		let environment = NSProcessInfo.processInfo().environment
+		if let xcodeVersionActual = environment["XCODE_VERSION_ACTUAL"] as? String {
```

Comment:
I used an environment file that is pretty Xcode specific, should not be set in any other environment. Suggestions welcome